wca-guidelines.md

```diff
@@ -72,7 +72,7 @@ To be more informative, each Guideline is classified using one of the following
 - 3h++) [EXAMPLE] Examples of enhancements include: new moves are possible, normal moves are impossible, more pieces or faces are visible, colors on the backside of the puzzle are visible, moves are done automatically, or the puzzles have more/different solved states.
 - 3h2++) [CLARIFICATION] "Stickerless" puzzles that significantly differ from most mass-produced "stickerless" puzzles are only permitted at the discretion of the WCA Delegate.
 - 3j+) [REMINDER] In the past, engraved/embossed parts have been permitted. This is no longer permitted.
-- 3j++) [CLARIFICATION] On Clock, pins must not be distinguishable from any other pin of the same side.
+- 3j++) [CLARIFICATION] A clock with distinguishable pins from any other pin of the same side is permitted.
```

Comment:
This isn't the PR for Clock pins @hanwu85

Reply:
This is strange, I didn't change this one...Branch mixed up I think.